src/main/java/com/codeborne/selenide/SelenideConfig.java

```diff
@@ -4,25 +4,22 @@
 import org.openqa.selenium.remote.DesiredCapabilities;
 
 import static com.codeborne.selenide.AssertionMode.STRICT;
-import static com.codeborne.selenide.Browsers.FIREFOX;
+import static com.codeborne.selenide.Browsers.CHROME;
 import static com.codeborne.selenide.FileDownloadMode.HTTPGET;
 import static com.codeborne.selenide.SelectorMode.CSS;
 
 public class SelenideConfig implements Config {
-  private String browser = System.getProperty("selenide.browser", System.getProperty("browser", FIREFOX));
+  private String browser = System.getProperty("selenide.browser", System.getProperty("browser", CHROME));
   private boolean headless = Boolean.parseBoolean(System.getProperty("selenide.headless", "false"));
   private String remote = System.getProperty("selenide.remote", System.getProperty("remote"));
-  private String browserSize = System.getProperty("selenide.browserSize", System.getProperty("selenide.browser-size"));
-  private String browserVersion = System.getProperty("selenide.browserVersion",
-    System.getProperty("selenide.browser.version", System.getProperty("browser.version")));
-  private String browserPosition = System.getProperty("selenide.browserPosition");
-  private boolean startMaximized = Boolean.parseBoolean(System.getProperty("selenide.startMaximized",
-    System.getProperty("selenide.start-maximized", "true")));
+  private String browserSize = System.getProperty("selenide.browserSize", "1024x768");
```

Comment:
maybe 1366x768 default will be better? 
See statistics https://www.w3schools.com/browsers/browsers_display.asp

Reply:
This value based on note from #810:
> I recommend to set browser window size explicitly by using Configuration.browserSize=1024x768 (a minimal browser size that your application should support).

Reply:
yep I got it. But I think `1024x768` is legacy screen resolution

Reply:
Yes, I put `1024x768` without any reason. Just some memories from my childhood. :)